src/files.js

```diff
@@ -491,6 +492,8 @@ FOAM_FILES([
   { name: "foam/net/auth/TokenBearerHTTPRequest" },
   { name: "foam/net/auth/AuthAgent" },
   { name: "foam/doc/ClassLink", flags: ['debug', 'web'] },
+  { name: "foam/doc/CodeTab", flags: ['debug', 'web'] },
```

Comment:
Be sure to remove these before committing. The only thing you should need to add to files.js is the UnstyledTabs